installer/manifests/keptn/charts/control-plane/templates/_helpers.tpl

```diff
@@ -263,4 +260,17 @@ securityContext:
 {{- end -}}
 {{- end -}}
 
+{{- define "control-plane.common.update-strategy" -}}
+{{- if (.Values.common).strategy -}}
+strategy:
+{{- toYaml .Values.common.strategy | nindent 4 -}}
```

Comment:
shouldn't the indentation happen in the consuming manifest file instead of in the template directly?